dev/io.openliberty.microprofile.metrics.internal.cdi.3.0/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>com.ibm.ws.microprofile.metrics.3.0.monitor</name>
+	<name>io.openliberty.microprofile.metrics.internal.cdi.3.0</name>
```

Comment:
This doesn't look right.  Rename from `com.ibm.ws.microprofile.metrics.3.0.monitor` to `io.openliberty.microprofile.metrics.internal.cdi.3.0`?